sdk/translation/azure-ai-translation-text/tests/test_break_sentence_async.py

```diff
@@ -0,0 +1,86 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from devtools_testutils import recorded_by_proxy
+from azure.ai.translation.text.models import InputTextItem
+from preparer import TextTranslationPreparer
+from testcase import TextTranslationTest
+import asyncio
+
+class TestBreakSentenceAsync(TextTranslationTest):
+
+    @TextTranslationPreparer()
+    @recorded_by_proxy
+    def test_autodetect(self, **kwargs):
+        async def test_autodetect_async(**kwargs):
+            endpoint = kwargs.get("text_translation_endpoint")
+            apikey = kwargs.get("text_translation_apikey")
+            region = kwargs.get("text_translation_region")
+            client = self.create_async_client(endpoint, apikey, region)
+            input_text_elements = [InputTextItem(text="Hello world")]
+
+            response = await client.find_sentence_boundaries(content=input_text_elements)
+            assert response is not None
+            assert response[0].detected_language.language == "en"
+            assert response[0].detected_language.score == 1
+            assert response[0].sent_len[0] == 11
+        asyncio.run(test_autodetect_async(**kwargs))
```

Comment:
If you're using the `AzureRecordedTestCase` class as a base for your `TextTranslationTest` you dont need to nest your test or have the `asyncio.run` for async tests, you can simply define them like in the example I'm sharing below and `AzureRecordedTestCase` will run them correctly for you. 

Example: 
```suggestion
    @recorded_by_proxy_async
    async def test_autodetect_async(**kwargs):
            endpoint = kwargs.get("text_translation_endpoint")
            apikey = kwargs.get("text_translation_apikey")
            region = kwargs.get("text_translation_region")
            client = self.create_async_client(endpoint, apikey, region)
            input_text_elements = [InputTextItem(text="Hello world")]

            response = await client.find_sentence_boundaries(content=input_text_elements)
            assert response is not None
            assert response[0].detected_language.language == "en"
            assert response[0].detected_language.score == 1
            assert response[0].sent_len[0] == 11
```

Reply:
Here are more docs on how to configure tests: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#the-devtools_testutils-package

Reply:
Also note that I added a line: `async with client:`

Reply:
That's how I originally coded the tests, but they failed with an error about the coroutine type.  I tried implementing one test using your provided template and it failed again.  My client is in fact descended from AzureRecordedTestCase.

Reply:
New commit with updated tests using provided format & pushed updated assets.json file.